servers/plugins/org.fusesource.ide.server.karaf.core/plugin.xml

```diff
@@ -68,319 +68,320 @@
+       			<facet id="jst.camel" version="2.15"/>
+     		</supported>
+   			<supported>
+        		<runtime-component
+         			id="org.fusesource.ide.karaf.runtime.component"
+         			version="2.3"/>
+                <facet id="wst.web"  version="1.0,1.2,1.3,1.4"/>
+       			<facet id="jst.web"  version="2.2,2.3,2.4,2.5,3.0"/>
+       			<facet id="jst.java" version="1.3,1.4,5.0,6.0,1.7,1.8"/>
+       			<facet id="jst.utility" version="1.0"/>
+       			<facet id="jst.camel" version="2.15"/>
+     		</supported>
+   			<supported>
+        		<runtime-component
+         			id="org.fusesource.ide.karaf.runtime.component"
+         			version="2.4"/>
+                <facet id="wst.web"  version="1.0,1.2,1.3,1.4"/>
+       			<facet id="jst.web"  version="2.2,2.3,2.4,2.5,3.0"/>
+       			<facet id="jst.java" version="1.3,1.4,5.0,6.0,1.7,1.8"/>
+       			<facet id="jst.utility" version="1.0"/>
+       			<facet id="jst.camel" version="2.15"/>
+     		</supported>
+   			<supported>
+        		<runtime-component
+         			id="org.fusesource.ide.karaf.runtime.component"
+         			version="3.0"/>
+                <facet id="wst.web"  version="1.0,1.2,1.3,1.4"/>
+       			<facet id="jst.web"  version="2.2,2.3,2.4,2.5,3.0"/>
+       			<facet id="jst.java" version="1.3,1.4,5.0,6.0,1.7,1.8"/>
+       			<facet id="jst.utility" version="1.0"/>
+       			<facet id="jst.camel" version="2.15"/>
+     		</supported>
+   </extension>  
+   
+   <extension
+         point="org.eclipse.wst.common.project.facet.core.runtimes">
+     <adapter>
+      <runtime-component id="org.fusesource.ide.karaf.runtime.component" version="2.2"/>
+      <factory class="org.fusesource.ide.server.karaf.core.runtime.classpath.KarafClasspathProviderAdapterFactory"/>
+      <type class="org.eclipse.jst.common.project.facet.core.IClasspathProvider"/>
+	</adapter>
+     <adapter>
+      <runtime-component id="org.fusesource.ide.karaf.runtime.component" version="2.3"/>
+      <factory class="org.fusesource.ide.server.karaf.core.runtime.classpath.KarafClasspathProviderAdapterFactory"/>
+      <type class="org.eclipse.jst.common.project.facet.core.IClasspathProvider"/>
+	</adapter>
+     <adapter>
+      <runtime-component id="org.fusesource.ide.karaf.runtime.component" version="2.4"/>
+      <factory class="org.fusesource.ide.server.karaf.core.runtime.classpath.KarafClasspathProviderAdapterFactory"/>
+      <type class="org.eclipse.jst.common.project.facet.core.IClasspathProvider"/>
+	</adapter>
+     <adapter>
+      <runtime-component id="org.fusesource.ide.karaf.runtime.component" version="3.0"/>
+      <factory class="org.fusesource.ide.server.karaf.core.runtime.classpath.KarafClasspathProviderAdapterFactory"/>
+      <type class="org.eclipse.jst.common.project.facet.core.IClasspathProvider"/>
+	</adapter>
+   </extension>
+   
+   
+  <!-- 		NEW_SERVER_ADAPTER add id for new runtime in the runtimeIds field below -->
```

Comment:
And no, please do not fully delete it as we will bring that back next version and I am too lazy to look it up from an older state of the 8.0.0 branch then.